diff -urb a/js/ui/background.js b/js/ui/background.js
--- a/js/ui/background.js	2019-10-11 21:57:16.736144726 +0300
+++ b/js/ui/background.js	2019-10-11 22:07:38.356612421 +0300
@@ -137,6 +137,7 @@
         this._fileMonitors = {};
         this._backgroundSources = {};
         this._animations = {};
+        this._clock = null;
     }
 
     monitorFile(file) {
@@ -189,6 +190,12 @@
         });
     }
 
+    getWallClock() {
+        if (!this._clock)
+            this._clock = new GnomeDesktop.WallClock();
+        return this._clock;
+    }
+
     getBackgroundSource(layoutManager, settingsSchema) {
         // The layoutManager is always the same one; we pass in it since
         // Main.layoutManager may not be set yet
@@ -242,7 +249,8 @@
         this._cancellable = new Gio.Cancellable();
         this.isLoaded = false;
 
-        this._clock = new GnomeDesktop.WallClock();
+        let cache = getBackgroundCache();
+        this._clock = cache.getWallClock();
         this._timezoneChangedId = this._clock.connect('notify::timezone',
             () => {
                 if (this._animation)
